serverless-inference/serverless-model-registry.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>Any doc to link listing these options?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>